README.md

```diff
@@ -1,2 +1,70 @@
-# math-magicians
-"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) developed using React that allows users to perform mathematical operations
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
```

Comment:
- [ ] please add details to your readme file, and also please remove the second readme file.